index.js

```diff
@@ -5,6 +5,7 @@ export const startEvaluate = ({
   ieContract,
   ieContractWithSigner,
   web3Storage,
+  record,
   logger
 }) => {
```

Comment:
I feel the name `record` is a bit ambiguous. Possible alternatives:
- `recordTelemetry`
- `recordTelemetryPoint`
- `telemetryPoint`

Thoughts?

Reply:
I was on the fence with the name too. I like the ones starting with a verb like `record` and `track`, because the fits it being a function. Feel free to pick one 👍